mgz/header/objects.py

```diff
@@ -110,6 +110,7 @@
         If(lambda ctx: find_save_version(ctx) >= 20.16, Struct(
             "peek"/Peek(Bytes(6)),
             If(lambda ctx: find_save_version(ctx) >= 25.22 and (find_type(ctx) == 10 or find_save_version(ctx) >= 63.0), Bytes(1)),
+            If(lambda ctx: find_save_version(ctx) >= 64.3, Bytes(1)),
```

Comment:
Is this byte actually here? I would have guessed it'd be after line 118. Probably does not matter in practice though.

Reply:
I did a diff between an old patch object and a new patch object of the same type. There is an additional byte, that looks like a flag, next to 2 other bytes/flags. Whether it was put in front or at the end is anyone's guess, since we don't currently know what these flags indicate